mlrun/feature_store/steps.py

```diff
@@ -431,45 +426,42 @@ def _do_pandas(self, event):
 
 
 class SetEventMetadata(MapClass):
-    """Set the event metadata (id, key, timestamp) from the event body"""
+    """Set the event metadata (id and key) from the event body"""
 
     def __init__(
         self,
         id_path: str = None,
         key_path: str = None,
-        time_path: str = None,
```

Comment:
This breaks backwards compatibility - if we read a persisted graph that used this step and the `time_path` parameter, it will fail loading as this attribute no longer exists. For now, should keep this parameter and make it deprecated -> not use it, and provide a deprecation warning.

Reply:
Alright.